src/cli/Generate.cpp

```diff
@@ -107,7 +107,7 @@ int Generate::execute(const QStringList& arguments)
         classes |= PasswordGenerator::EASCII;
     }
 
-    PasswordGenerator::GeneratorFlags flags = 0x0;
+    PasswordGenerator::GeneratorFlags flags = nullptr; // 0x0
```

Comment:
Same as above

Reply:
I'll be honest, I wasn't quite sure of this change but the static analyzer reported it as initialization of a pointer.
I guess the reason is that QFlags has this overloaded constructor:
```cpp
QFlags(QFlags::Zero = Q_NULLPTR)
```
I'm gonna changing it back because `0x0` was also more readable, thinking of it.